### ISSUE TITLE
Attribute + License Logo

### ISSUE THREAD

**Issue by cschroed-usgs** [cf-convention/cf-convention.github.io#64]
Hi There! Is this the source for the globe logo being used on the GitHub Pages CF site?
https://commons.wikimedia.org/wiki/File:Simple_Globe.svg

If so, would you consider declaring the logo attribution and license so that the CF site's usage is more compliant with the author's license?

If not, is there some other license or attribution for the logo that needs to be declared?

**Comment by taylor13** [comment]
Anyone know the origin of the logo on the CF website?

**Comment by cofinoa** [comment]
I have added .png and .svg banner logos in the PR #115 and they have been used for the registration form for the 2020 CF Workshop

**Comment by erget** [comment]
@cofinoa already implemented a fix for this: https://github.com/cf-convention/cf-convention.github.io/issues/116#issuecomment-638887223

Would you mind wrapping that into a PR and associating it to this issue? Attribution is important, so I feel bad about letting that sit around for so long.

**Comment by JonathanGregory** [comment]
I agree that attribution is important, but I feel it is a bit disproportionate to put a lengthy attribution phrase for the globe, which isn't really essential to CF, in the footer which will appear on every page. Could we put the attribution in text which would appear if you hover on the globe instead? The author says any reasonable method is OK.
